pkg/operator/api/api.go

```diff
@@ -1,4 +1,6 @@
 /*
+Package api for the operator api manager.
```

Comment:
can this comment move down to just the `package` statement? It would be easier to no disturb the copyright and will make search and replace easier in the future.